pkgs/tools/system/auto-cpufreq/default.nix

```diff
@@ -25,15 +30,18 @@ python3Packages.buildPythonPackage rec {
 
     # patch to prevent script copying and to disable install
     ./prevent-install-and-copy.patch
+
+    # patch to remove venvs from the service file
+    ./remove-service-venv.patch
```

Comment:
Are there any upstream issues or PR related to these patches?

Reply:
There are issues complaining about venv stuff, but they're mostly resolved by users switching to the Snap package.

Basically this is just an artifact of how auto-cpufreq is typically installed that prevents us from trivially upgrading from version 1.7.1.  As far as I can tell - it's not necessary _except_ to reduce the dependency on the system python version.